media-sound/amarok/amarok-9999.ebuild

```diff
@@ -48,6 +52,7 @@ DEPEND="
 	>=kde-frameworks/kitemviews-${KFMIN}:5
 	>=kde-frameworks/knewstuff-${KFMIN}:5
 	>=kde-frameworks/knotifications-${KFMIN}:5
+	>=kde-frameworks/knotifyconfig-${KFMIN}:5
```

Comment:
I suspect it is a bogus dependency. Even though listed in CMakeLists.txt, a quick check revealed no linking against it.

Reply:
https://invent.kde.org/multimedia/amarok/-/merge_requests/26